### DISCUSSION TITLE
graphql chaining

### DISCUSSION THREAD

**Discussion by jbockle** [ChilliCream/hotchocolate#4683] [Q&A] [answered]
If you had multiple services exposing graphql, and wanted to integrate a fetch source in one to another is there an existing integration for that? I get I could use strawberry shake, but if I didn't want to over/under fetch from the target how would I do that?

**Comment by michaelstaib** [comment, marked as answer]
HotChocolate.Stitching could be a solution depending on your scenario.

**Reply by jbockle** [reply]
Sorry I must've overlooked that in the docs! That looks like exactly what I'm looking for.
